src/Game/CritterObject.cpp

```diff
@@ -290,6 +290,21 @@ namespace Falltergeist
             _traitsTagged.at((unsigned)num) = value;
         }
 
+        void CritterObject::setCritterFlags(unsigned int flags)
```

Comment:
this is cumbersome code.

Reply:
It's done this way in `Object::setFlags` as well. I wouldn't mind rewriting the code to keep flags in a single variable, but my main intention is to keep the style the way @alexeevdv wants it.

Refactoring code like this in the future would not be a big issue. The main problem we currently face IMHO is how to get rid of all those raw pointers in the engine.